App/index.js

```diff
@@ -0,0 +1,21 @@
+function unhidePlaylist(id){
+    wantedId = null;
```

Comment:
Declare ALL vars at the start of the function. Always use "var" when declaring variables